### PR TITLE
Final edits

### PR DESCRIPTION
**Pull request by zachary-foster**
I looked over everything on this repository, but not the example repositories. I rephrased and reformatted a few things; change em back if you disagree.

### CODE REVIEW

File: 02--Setup.Rmd [hunk 1]
```diff
@@ -11,27 +11,29 @@
...
    Desktop](https://www.rstudio.com/products/rstudio/download/) integrated 
-   development environment (IDE) that we recommend.
+   development environment (IDE), which we recommend. This is basically a point-and-click
+   interface for R that includes a text editor, file browser, and some other conveniences.
 
 Installing the required packages
 ----
 
-The following packages are utilized in this primer:
+The following packages are used in this primer:
```
Comment (zkamvar) [inline 02--Setup.Rmd:39]: Good catch!

In almost all cases used >>> utilized.

File: 02--Setup.Rmd [hunk 2]
```diff
@@ -11,27 +11,29 @@
 title: "Required Components"
 ---
 
-This guide is a brief introduction to some of the principles of reproducible
-science in R. Before we get started, we should install the necessary components
-for this tutorial.
+This website is an introduction to some of the principles of reproducible
+science in R, not R itself or associated software and extensions (i.e. "packages").
```
Comment (zkamvar) [inline 02--Setup.Rmd]: This sounds like legalese. 😧

File: tutorial.css [hunk 3]
```diff
@@ -80,9 +80,9 @@ in the center, so it's placed to the left. These are used with .knit_border and
 
 #exercise {
   display: block;
-  width: 98%;
+  width: 100%;
```
Comment (zkamvar) [inline tutorial.css:83]: I placed this at 98% because the iframe was cutting off the border at 100%

Comment (zkamvar) [inline tutorial.css:83]: Just tried building the website with quilt(). 

Apparently this works. I wonder why it didn't before

**Comment by zkamvar** [comment]
I've clarified the setup chapter a bit more. To do this, I'm using the printr package: https://github.com/yihui/printr, which I think is great!
- utilize printr for displaying a help page
- change example from getwd() to sessionInfo()
  IMO, this is MUCH more useful than getwd() in
  the context of reproducible research
- change help topic from poppr to tidyr
  Not everyone is a population geneticist, but
  everyone could use tidy data!
- display the help page for tidyr's gather
- add an example of how to see examples :)

What do you think? If you like it, merge it.

File: 02--Setup.Rmd (hunk 2 above)
Comment (zachary-foster) [inline 02--Setup.Rmd]: Yea, your version is better

File: tutorial.css (hunk 3 above)
Comment (zachary-foster) [inline tutorial.css:83]: Hmm, not sure why that would be. I have notice odd stuff occasionally, but there should be JavaScript to scale the iframes automatically.

**Comment by zachary-foster** [comment]
I think all that looks good. The `printr` package looks cool. Good stuff.

**Comment by zkamvar** [comment]
Would you like me to push the gh-pages branch?

**Comment by zachary-foster** [comment]
Yea, I still cant get the image css to apply correctly. Thanks

**Pull request merged**